domains/artworld/domain.yaml

```
@@ -14,30 +24,293 @@ attributes:
+      mutation {
+        defineArtworkDetailsEntity(
+          externalId: "salvatormundidetails"
+          attributes: {
+            titleAttribute: "Salvator Mundi"
+            descriptionAttribute: "Depiction of Christ holding a crystal orb and making the sign of the blessing."
+          }
+        ) {
+          context
+          txId
+        }
+      }
+      ```
+
     attributes:
       - Title
       - Description
 activities:
   Exhibited:
+    doc: |
+      # `Exhibited`
+
+      `Exhibited` refers to the display of an artwork in a public space.
+
+      ## Examples
+
+      ```graphql
+      mutation {
+        defineExhibitedActivity(
+          externalId: "salvatormundiprado"
+          attributes: {
+            locationAttribute: "Prado Museum"
+          }
+        ) {
+          context
+          txId
+        }
+      }
+      ```
+
     attributes:
       - Location
   Created:
+    doc: |
+      # `Created`
+
+      `Created` refers to the act of creating a new piece of art.
+
+      ## Examples
+
+      When a new artwork is created, this event can be recorded as a `Created` activity,
+      with the artist and the title of the artwork both noted.
+
+      ```graphql
+      mutation {
+        defineCreatedActivity(
+          externalId: "salvatormundi"
+          attributes: { titleAttribute: "Salvator Mundi" }
+        ) {
+          context
+          txId
+        }
+      }
+      ```
+
+      Recording Associations of Artworld Agents with Artworld Activities and the Output
+
+      ```graphql
+      mutation {
+        wasAssociatedWith(
+          responsible: { externalId: "artist001" }
+          activity: { externalId: "salvatormundi" }
+          role: CREATOR
+        ) {
+          context
+          txId
+        }
+      }
+      ```
+
     attributes:
       - Title
   Sold:
+    doc: |
+      # `Sold`
+
+      `Sold` refers to the transaction between a collector and an artist,
+      in which the collector purchases the artwork for a certain amount of money.
+
+      ## Examples
+
+      When an artwork is sold, this can be recorded as a `Sold`` activity,
+      along with details of the buyer and the purchase price.
+
+      ```graphql
+      mutation {
+        defineSoldActivity(
+          externalId: "saleofsalvatormundi"
+          attributes: {
+            purchaseValueAttribute: "450000000"
+            purchaseValueCurrencyAttribute: "USD"
+          }
+        ) {
+          context
+          txId
+        }
+      }
+      ```
+
+      Recording Associations of Artworld Agents with Artworld Activities and the Output
+
+      ```graphql
+      mutation {
+        wasAssociatedWith(
+          responsible: { externalId: "collector471" }
+          activity: { externalId: "saleofsalvatormundi" }
+          role: SELLER
+        ) {
+          context
+          txId
+        }
+      }
+      ```
+
+      One could query the record to determine the average sale price of artworks
```

```suggestion
      One could query the record to determine the sale prices of artworks
```
(seeing as the query doesn't average for us)